### ISSUE TITLE
No support for #define in scripts

### ISSUE THREAD

**Issue by faissaloo** [IsmAvatar/LateralGM#307]
In GM for some reason some scripts will be packaged together in text files, with each script separated by:  
```
#define <Script name>
```  
  
LateralGM should expand these when the file is opened.

**Comment by RobertBColton** [comment]
That's probably the case when bookmarks are used, so for that, LGM would just have to straight up add bookmark support.

**Comment by RobertBColton** [comment]
I am just posting this here to show @JoshDreamland how the feature works.

You create the tabbed scripts:
![Tabbed Script](https://cloud.githubusercontent.com/assets/3212801/19011124/6f0e83e8-875b-11e6-904d-56abdbda71a4.png)

This produces the following text file:
```
#define script0
// script 0

#define script1
// script 1

#define script2
// script 2
```

**Comment by RobertBColton** [comment]
I have fixed this in #315 but there is not yet any tab support in LGM. It will just split the scripts up when reading a GMX. At some point in the future we can make the script editor tabbed or something and also take care of #102 at the same time since it is all pretty much related.
